invokeai/app/api/routers/model_manager.py

```diff
@@ -630,7 +630,13 @@ async def convert_model(
         raise HTTPException(400, f"The model with key {key} is not a main checkpoint model.")
 
     # loading the model will convert it into a cached diffusers file
-    model_manager.load.load_model(model_config, submodel_type=SubModelType.Scheduler)
+    try:
+        cc_size = loader._convert_cache.max_size
+        if cc_size == 0:  # temporary set the convert cache to a positive number so that cached model is written
+            loader._convert_cache.max_size = 1.0
+        loader.load_model(model_config, submodel_type=SubModelType.Scheduler)
+    finally:
+        loader._convert_cache.max_size = cc_size
```

Comment:
`_convert_cache` isn't on the ABC so accessing it is unsafe

Reply:
Fair enough. Fixed.

Reply:
Not safe to assign to an unknown attr: `loader._convert_cache.max_size = 1.0`